railties/lib/rails/application/configuration.rb

```diff
@@ -261,6 +261,8 @@ def load_defaults(target_version)
         when "7.1"
           load_defaults "7.0"
 
+          self.add_autoload_paths_to_load_path = false
```

Comment:
Update the configuring guide 😂

Reply:
Arf, everytime I miss one -_-

Reply:
Actually it's already documented.

Reply:
I mean document that in 7.1 the default will be false. But I guess we need to merge #44120 first.